src/index.ts

```diff
@@ -18,6 +15,12 @@ const uuid = (len: number) =>
       return;
     }
 
+    const isBrowser = typeof globalThis.crypto !== 'undefined';
```

Comment:
This is not sufficient to test a browser. 

Node.js has crypto too https://nodejs.org/api/globals.html#crypto_1

Reply:
I think you should change this to `if (globalThis?.crypto?.getRandomValues === 'function')`